src/common/queries/useBurnBlock.ts

```diff
@@ -8,9 +8,20 @@ import {
 import { BurnBlock } from '@stacks/blockchain-api-client';
 
 import { useApi } from '../api/useApi';
+import { useGlobalContext } from '../context/useAppContext';
 
 export const BURN_BLOCKS_QUERY_KEY = 'burnBlocks';
 
+export function useGetBurnBlockQuery() {
+  const { url: activeNetworkUrl } = useGlobalContext().activeNetwork;
+  return (heightOrHash: string | number) => ({
+    queryKey: [BURN_BLOCKS_QUERY_KEY, heightOrHash],
+    queryFn: () =>
+      fetch(`${activeNetworkUrl}/extended/v2/burn-blocks/${heightOrHash}`).then(res => res.json()),
```

Comment:
Here also we can use BurnBlocksApi.getBurnBlock

Reply:
done